edx_exams/apps/core/api.py

```diff
@@ -26,6 +33,22 @@ def get_attempt_by_id(attempt_id):
 
     return None
 
+def get_attempt_by_user_id(user_id):
```

Comment:
I think this should have a more descriptive name, like `get_attempts_in_progress`, as opposed to just by user ID